### ISSUE TITLE
Pause & Resume

### ISSUE THREAD

**Issue by alx-xlx** [unamer/IDMHelper#1]
is there any future for pause and resume using cli

**Comment by rip1s** [comment]
nope, idm doesn't support such api, but u can always locate idm window and sendmessage to pause button yourself.
